parts/k8s/cloud-init/artifacts/label-nodes.service

```diff
@@ -5,4 +5,6 @@ After=kubelet.service
 Restart=always
 RestartSec=60
 ExecStart=/bin/bash /opt/azure/containers/label-nodes.sh
+[Install]
```

Comment:
@mboersma FYI, these missing lines were preventing `systemctl enable` from working (and a systemd bug that returned 0 upon failure was hiding this from us)